llvm/test/CodeGen/PowerPC/aix-small-tls-globalvarattr-loadaddr.ll

```diff
@@ -0,0 +1,221 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc  -verify-machineinstrs -mcpu=pwr7 -ppc-asm-full-reg-names \
+; RUN:      -mtriple powerpc64-ibm-aix-xcoff < %s \
```

Comment:
nit: the default value of `aix-small-local-exec-tls` is false, in case of we change the default value of `aix-small-local-exec-tls` to true.  please add `-mattr=-aix-small-local-exec-tls` here ,so we do not need to modify the test case.